vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -3,15 +3,11 @@
         class="select-checkboxes"
         v-bind:options="selectOptions"
         v-bind:value="'checkbox-group'"
-        v-bind:disabled="disabled"
-        v-bind="{
-            autoScroll: false,
-            maxHeight: 210,
-            ...selectProps
-        }"
+        ref="select"
+        v-on:select:keydown="scrollToInput"
     >
         <template v-slot:checkbox-group>
-            <div class="checkboxes" v-on:click.stop>

```

Comment:
Why did you remove this? This can break things in other places.

Reply:
It was meant to test things, as I was having difficulties getting the input. Forgot to put it back, similar to the `bind` and `disabled` parameters. Already re-added them, thanks!